modules/components/src/Aggs/TermAgg.js

```diff
@@ -10,6 +10,7 @@ const TermAggs = ({
   handleValueClick = () => {},
   isActive = () => {},
   Content = 'div',
+  defaultMaxEntries = 5,
```

Comment:
maybe this props should be called `maxTerms`. the `5` is the default part, the `prop` itself isn't default